Assets/Scripts/Utilities/AudioZone.cs

```diff
@@ -0,0 +1,31 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class AudioZone : MonoBehaviour 
+{
+	[SerializeField] AudioSource sourceData = null;
+	AudioSource playingClip = null;
+
+	void OnTriggerEnter( Collider col )
+	{
+		if ( col.GetComponent<PlayerActor>() )
+		{
+			playingClip = SoundManager.instance.Play3DSoundAndFollow( sourceData, transform );
+		}
+	}
+
+	void OnTriggerExit( Collider col )
```

Comment:
Hmm, that's a fair point. Something like `otherCol` or even just `other` might be better, though.